locales/en.json

```diff
@@ -190,7 +190,20 @@
             "invalid_year": "The card's expiration year is invalid.",
             "invalid_month": "The card's expiration month is invalid.",
             "invalid_cvv": "The card's CVV is invalid.",
-            "expired": "The card's expiration date is passed"
+            "expired": "The card is expired.",
+            "declined": "The card has been declined. Please call your card issuer for more information.",
+            "address_invalid": "The card’s address is incorrect.",
+            "insufficient_funds": "The account did not have sufficient funds to cover the transaction amount.",
+            "square": {
+                "card_not_supported": "The card is not supported in the geographic region."
+            },
+            "paypal": {
+                "currency_not_supported": "The currency is not currently supported by PayPal."

```

Comment:
Does this mean that the currency is not supported by PayPal, or that the card used by the customer can't process a transaction in the requested currency?

Reply:
https://www.paypal.com/us/smarthelp/article/why-did-i-get-api-error-code-10755-ts1128 paypal